debian/ceph-base.install

```diff
@@ -5,6 +5,7 @@ usr/bin/ceph-run
 usr/bin/crushtool
 usr/bin/monmaptool
 usr/bin/osdmaptool
+usr/bin/ceph-kvstore-tool
```

Comment:
@smithfarm could you please take this chance to add `usr/share/man/man8/ceph-kvstore-tool.8` to this list?